app/controllers/games_controller.rb

```diff
@@ -34,11 +34,38 @@ def update
 
   private
 
-  helper_method :current_game
+  helper_method :current_game, :place_piece_td
   def current_game
     @current_game ||= Game.find(params[:id])
   end
 
+  def board_display_piece_query(row, column)
+    current_game.pieces.find do |f|
+      f["x_position"] == column && f["y_position"] == row
+    end
+  end
+
+  def place_piece_td(row, column)
+    find_piece = board_display_piece_query(row, column)
+    board_square = "<td class='y-position-'#{column}' "
+    board_square += "piece-id-data='#{piece_id(find_piece)}' "
+    board_square += "piece-type-data='#{piece_type(find_piece)}''>"
+    unless find_piece.nil?
+      board_square += ActionController::Base.helpers.image_tag find_piece
+                      .image_name, size: '40x45',
+                                   class: 'img-responsive center-block'
+    end
+    board_square + "</td>"
+  end
```

Comment:
this method generates an html td element with tags for the piece id and piece type (and column) along with an image_tag to associate the correct image with the piece and its color